test/e2e/TestScheduleTx.ts

```diff
@@ -26,7 +26,7 @@ export const getHelperMethods = (web3: any) => {
 
   function waitUntilBlock(seconds: any, targetBlock: any) {
     return new Promise(resolve => {
-      const asyncIterator = function asyncIterator() {
+      const asyncIterator = function asyncIte() {
```

Comment:
Instead of `asyncIte` which is unclear I think it could be better to use `_asyncIterator`.